lib/segment/src/vector_storage/quantized/quantized_multi_query_scorer.rs

```diff
@@ -74,9 +76,11 @@ where
     type TVector = [TElement];
 
     fn score_stored(&self, idx: PointOffsetType) -> ScoreType {
-        let sub_vectors_count = self.quantized_multivector_storage.get_offset(idx).count as usize;
+        let multi_vector_offset = self.quantized_multivector_storage.get_offset(idx);
+        let sub_vectors_count = multi_vector_offset.count as usize;
         self.hardware_counter.vector_io_read().incr_delta(
-            self.quantized_multivector_storage.quantized_vector_size() * sub_vectors_count,
+            size_of::<MultivectorOffset>()
```

Comment:
`hardware_counter` has `hardware_counter.set_cpu_multiplier(size_of::<TElement>());` so MultivectorOffset may be over-counted.